js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -58,70 +61,102 @@ export const LaunchAssetExecutionButton: React.FC<{
       disabledReason || 'Assets must be in the same job to be materialized together.';
   }
 
+  const assetsWithConfig = assets.filter((a) => configSchemaForAssetNode(a));
```

Comment:
If this is just for the check on 65, you can actually do `assets.some(a=>configSchemaForAssetNode(a))` here for a tiny perf win!

Reply:
Done.